neqo-crypto/min_version.txt

```diff
@@ -1 +1 @@
-3.105
+3.110
```

Comment:
If this required for the test, or do you just want to keep us current?

Reply:
NSS crashes without the fix to https://bugzilla.mozilla.org/show_bug.cgi?id=1949677 which is in NSS 3.110.